### ISSUE TITLE
new `bibtex` command

### ISSUE THREAD

**Issue by benureau** [pubs/pubs#183]
Multiple time,s I have needed a quick way not to add a reference, but just display the bibtex of it, whether it is in the repository (`pubs bibtex Loeb_2012`) or not (`pubs bibtex -X 1803.10122`). A bibtex command would do that:
```shell
$ pubs bibtex -X 1803.10122
@article{https://doi.org/10.5281/zenodo.1207631,
  doi = {10.5281/zenodo.1207631},
  url = {https://zenodo.org/record/1207631},
  author = {Ha, David and Schmidhuber, Jürgen},
  title = {World Models},
  publisher = {Zenodo},
  year = {2018}
}
```

This would also make pubs relevant for people just interested by this function.
What do you think? I'll create the PR soon.

**Comment by omangin** [comment]
Yes, that is a good point. The difficulty is that it is overlapping with `list` and `export` (that were already overlapping with each other anyway). But maybe that is not such an issue as long as we correctly avoid code duplication.

In my case I thought multiple time that a `pubs show` command would be nice to have. I think this use case could fall under it. We could have `pubs show` default to printing metadata and bibdata in a pretty format (we can start with a yaml style list of attributes and make it prettier later). It would take either a citekey or an API source as in your example. There would not be metadata in the latter case. Then we can provide a `--bibtex` option to only output the bibtex.

```
$ pubs show -X 1803.10122
ArXiv document 1803.10122
  [not in local repository]
  Bibtex article
    title: World Models
    author: Ha, David and Schmidhuber, Jürgen
    publisher: Zenodo
    year: 2018
    doi: 10.5281/zenodo.1207631
    url: https://zenodo.org/record/1207631

$ pubs show Ha2018
Local paper Ha2018
  Added on Jan, 4th 2019
  Document: docsdir://Ha2018.pdf
  Tags: [Modelling]
  Bibtex article
    title: World Models
    author: Ha, David and Schmidhuber, Jürgen
    publisher: Zenodo
    year: 2018
    doi: 10.5281/zenodo.1207631
    url: https://zenodo.org/record/1207631
```
